### PR TITLE
Fix es-lint issues in ApiAvailability chart

### LINKED ISSUE #1014: Fix eslint errors availability chart
**Description:**
Fix eslint errors introduced in https://github.com/wso2/analytics-apim/pull/1011
Screen Shot 2020-02-27 at 12 56 35 PM

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by fazlan-nazeem**
fixes #1014

### CODE REVIEW

File: components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx [hunk 1]
```diff
@@ -176,28 +176,29 @@ class APIMApiAvailabilityWidget extends Widget {
...
+        const legendData = [{ name: 'Available' }, { name: 'Response time is high' },
+            { name: 'Server error occurred' }];
         if (data) {
-
             let availableCount = 0;
             let responseHighCount = 0;
             let serverErrorCount = 0;
             data.forEach((dataUnit) => {
                 if (dataUnit[0].includes(legendData[0].name)) {
+                    // eslint-disable-next-line prefer-destructuring
```
Comment (TanyaM) [inline components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: es-lint disabled?

Comment (fazlan-nazeem) [reply components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: I don't see a way how i can destructure this. any suggestions?

Comment (pahans) [reply components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: const [ availableCount, responseHighCount, serverErrorCount ] = dataModified;

Comment (fazlan-nazeem) [reply components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: wouldn't this mean

const availableCount = dataUnit[0];
const responseHighCount = dataUnit[1];
const serverErrorCount = dataUnit[2];

Comment (fazlan-nazeem) [reply components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: the intention is to always assign dataUnit[1]

Comment (pahans) [reply components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: `const [ ,serverErrorCount] = dataUnit;`

Comment (fazlan-nazeem) [reply components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: what about **availableCount**? it's value also should be dataUnit[1]

Comment (fazlan-nazeem) [reply components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: plus I need to use these variables outside the for loop scope. That's why I had to declare it before the for loop.

File: components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx [hunk 2]
```diff
@@ -176,28 +176,29 @@ class APIMApiAvailabilityWidget extends Widget {
...
                     availableCount = dataUnit[1];
```
Comment (pahans) [inline components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]:
```suggestion
                    const [ , availableCount] = dataUnit;
```

File: components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx [hunk 3]
```diff
@@ -176,28 +176,29 @@ class APIMApiAvailabilityWidget extends Widget {
...
-                } else if (dataUnit[0].includes(legendData[1].name)){ // Response time high messages are not unique
-                    let currentKeyResponseHighCount = dataUnit[1];
+                } else if (dataUnit[0].includes(legendData[1].name)) { // Response time high messages are not unique
+                    const currentKeyResponseHighCount = dataUnit[1];
                     responseHighCount += currentKeyResponseHighCount;
                 } else if (dataUnit[0].includes(legendData[2].name)) {
-                    serverErrorCount = dataUnit[1] ;
+                    // eslint-disable-next-line prefer-destructuring
+                    serverErrorCount = dataUnit[1];
```
Comment (pahans) [inline components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: const [ , serverErrorCount] = dataUnit;

File: components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx (hunk 1 above)
Comment (pahans) [reply components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: Added Code suggestions.

File: components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx (hunk 2 above)
Comment (fazlan-nazeem) [reply components/org.wso2.analytics.apim.widgets/APIMApiAvailability/src/APIMApiAvailabilityWidget.jsx]: With this change, I can't use availableCount or serverErrorCount outside the function scope.

**Pull request merged**